pyat/at/lattice/elements.py

```diff
@@ -972,6 +972,45 @@ def __init__(self, family_name: str, m66=None, **kwargs):
         super(M66, self).__init__(family_name, M66=m66, **kwargs)
 
 
+class SimpleQuantDiff(Element):
+    """Linear tracking element for simplified quantum diffusion"""
+    _BUILD_ATTRIBUTES = Element._BUILD_ATTRIBUTES + ['emit_x',
+                                                     'emit_y',
+                                                     'sigma_dp',
+                                                     'tau_x',
+                                                     'tau_y',
+                                                     'tau_z',
+                                                     'beta_x',
+                                                     'beta_y',
+                                                     'U0']
+
+    def __init__(self, family_name: str, emit_x: float, emit_y: float,
```

Comment:
Would it be simpler to default everything to None and use GetOptionalDouble in the C passmethod, this users only need to provide what is useful